src/components/SearchBar/SearchBar.jsx

```diff
@@ -14,27 +15,37 @@ export default function SearchBar() {
     <>
       <div className="searchBar__container">
         <div className="container searchBar__flex">
-          <form
-            style={!show ? { display: "none" } : undefined}
-            onSubmit={(e) => {
-              e.preventDefault();
-              navigate(`/productsByName/${product}`);
-              setProduct("");
-            }}
-            className="box searchBar"
-          >
-            <input
-              className="searchBar-textInput"
-              type="text"
-              value={product}
-              onChange={handleSubmit}
-              placeholder="Search products..."
-            />
-            <button type="submit" className="btn btn-primary">
-              Search
-            </button>
-          </form>
           <div className="searchBar__buttonPlace">
+            <div
+              className="btn-rounded searchBar__button searchBar__backButton"
+              onClick={() => {
+                window.history.back();
+              }}
```

Comment:
para controlar que usuario no se salga de nuestra app sugiero estos cambios. te dejo un ejemplo abajo... la idea en el else, seria q el botton se oculte

function returnBack(){

    // If last visited page before this page is in your domain
    if(document.referrer.indexOf(window.location.host) >= 0)
    {
        window.history.back()
    }
    else {
        // If last page is not in your domain, do your logic here..
    }

}